models/issue_tracked_time.go

```diff
@@ -20,7 +20,7 @@ type TrackedTime struct {
 	UserID      int64     `xorm:"INDEX"`
 	User        *User     `xorm:"-"`
 	Created     time.Time `xorm:"-"`
-	CreatedUnix int64     `xorm:"created"`
+	CreatedUnix int64     `xorm:"created_unix"`
```

Comment:
why do you rename the table column in db there is no need to do so

Reply:
I am not. Looks like this is an error in xorm.

This is what i have in sqlite db.
sqlite> .dump tracked_time
PRAGMA foreign_keys=OFF;
BEGIN TRANSACTION;
CREATE TABLE `tracked_time` (`id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, `issue_id` INTEGER NULL, `user_id` INTEGER NULL, `created_unix` INTEGER NULL, `time` INTEGER NOT NULL, `deleted` INTEGER DEFAULT 0 NOT NULL);

Reply:
The `created` here is a XORM keyword, not a column name.  The keyword shouldn't be changed.

So the column name is `CreatedUnix` => `created_unix` (implicitly), it was correct already.